tests/test_benchmarking.py

```diff
@@ -0,0 +1,44 @@
+"""Tests for config files in benchmarking/ folder."""
+import os
+import subprocess
+
+import pytest
+
+from imitation.data import types
+
+BENCHMARKING_DIR = types.parse_path("benchmarking")
+
+if not BENCHMARKING_DIR.exists():  # pragma: no cover
```

Comment:
I think we can get rid of this if we make the path relative to the script like https://github.com/HumanCompatibleAI/imitation/blob/master/tests/test_examples.py#L30

The other way I can think of is adding it as `package_data` in `setup.py` then using `pkg_resources` library -- but we probably don't actually want to bake these JSON configs into the wheel.